src/customers/service/customers.service.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { CustomersService } from './customers.service';
+import CustomersRepository from '../repository/customers.repository';
+import Customer from '../domain/customer.domain';
+
+describe('CustomersService', () => {
+  let customersService: CustomersService;
+  let customersRepository: jest.Mocked<CustomersRepository>;
+
+  beforeEach(async () => {
+    customersRepository = {
+      findOne: jest.fn(),
+      findOneByUsername: jest.fn(),
+      create: jest.fn(),
+    } as unknown as jest.Mocked<CustomersRepository>;
+
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [
+        CustomersService,
+        { provide: CustomersRepository, useValue: customersRepository },
+      ],
+    }).compile();
+
+    customersService = module.get<CustomersService>(CustomersService);
+  });
+
+  it('should be defined', () => {
+    expect(customersService).toBeDefined();
+  });
+
+  it('should create a customer', async () => {
+    const customer = {
+      id: '73ec7a0d-a499-4304-9629-670e5eebdc9d',
```

Comment:
Move `'73ec7a0d-a499-4304-9629-670e5eebdc9d'` into a variable and reuse it.

Reply:
done